src/upstart/ceph-rbd-mirror.conf

```diff
@@ -0,0 +1,26 @@
+description "Ceph rbd mirror"
+
+start on ceph-rbd-mirror
+stop on runlevel [!2345] or stopping ceph-rbd-mirror-all
+
+respawn
+respawn limit 3 1800
+
+limit nofile 8096 65536
```

Comment:
The systemd unit files have huge limits compared to the upstart files -- I copied this limit from radosgw since it was the only other "client".  The other daemons have "limit nofile 16384 16384" or "limit nofile 327680 327680" (OSD).